bai2.txt

```diff
@@ -0,0 +1 @@
+bai1-bai3
```

Comment:
- Nhận xét chung: code còn dơ
- Must: bắt buộc phải sửa
- Should: nên sửa -> ko sửa thì cũng merge dc
- Could: có thể sửa hoặc không sửa (VD: đổi đoạn code này thành thư viện thử)